dist/index.js

```diff
@@ -645,7 +645,7 @@ const windowsRelease = release => {
 	// Server 2008, 2012, 2016, and 2019 versions are ambiguous with desktop versions and must be detected at runtime.
 	// If `release` is omitted or we're on a Windows system, and the version number is an ambiguous version
 	// then use `wmic` to get the OS caption: https://msdn.microsoft.com/en-us/library/aa394531(v=vs.85).aspx
-	// If `wmic` is obsoloete (later versions of Windows 10), use PowerShell instead.
+	// If `wmic` is obsolete (later versions of Windows 10), use PowerShell instead.
```

Comment:
For my future reference:
https://github.com/sindresorhus/windows-release/blob/6f4a280599f8ac95725a35b471cfc0e2305aeb43/index.js#L32